packages/core/src/system.ts

```diff
@@ -34,6 +37,17 @@ export function createSystem<T extends ActorSystemInfo>(): ActorSystem<T> {
 
       keyedActors.set(systemId, actorRef);
       reverseKeyedActors.set(actorRef, systemId);
+    },
+    get deadLetters() {
+      if (!deadLetters) {
+        deadLetters = interpret(deadLettersBehavior, {
+          _system: system
+        });
+
+        deadLetters.start();
```

Comment:
It's started but isn't ever stopped. Shouldn't it become a regular child of the root to behave in the same way as its other child actors?